far/FarRus.hlf.m4

```diff
@@ -5643,75 +5643,89 @@ $ #far:config System.CASRule#
 
 @Panel.ShellRightLeftArrowsRule
 $ #far:config Panel.ShellRightLeftArrowsRule#
- Параметр позволяет управлять поведением стрелок влево/вправо (как на основной, так и на дополнительной клавиатуре).
-
- Значения:
- 0 - ^<wrap>поведение как у 1.70: если командная строка непустая, то клавиши
-Left/Right и Num4/Num6 действуют по-разному в зависимости от
-режима панели: если имена файлов отображаются в несколько колонок
-(по умолчанию режимы 2 и 3), то команды вправо/влево применяются
-к панели (как и при пустой командной строке); а если имена файлов
-отображаются в одну колонку (по умолчанию все остальные режимы),
-то команды вправо/влево применяются к командной строке.
- 1 - ^<wrap>клавиши Left/Right и Num4/Num6 при включённой панели всегда
-применяются только к ней, независимо от содержимого командной
-строки и режима панели.
-
- Примечание: в командной строке есть CtrlD/CtrlS.
+ Этот булевский параметр управляет поведением клавиш влево и вправо,
```

Comment:
`булевский`, not `булевый`? (I'm not an expert)

Reply:
Neither am I. [Wikipedia](https://ru.wikipedia.org/wiki/Логический_тип) says "булев" or "булевый". MS Word accepts both "булев" and "булевский", but not "булевый". Let's settle on "булев" if you are OK with this.

Reply:
Maybe `логический` then, as in the title?
Also, when in doubt, I usually check Microsoft Terminology - [they prefer "логический" too](https://www.microsoft.com/en-us/language/Search?&searchTerm=boolean&langID=635&Source=true&productid=0).

Reply:
Interesting place; thank you for the pointer. After some thought, I rather like "логический" too. Nevertheless, I asked around. Somebody suggested "логический (Boolean)" to disambiguate. What do you think?

I also posted the question in a very relevant community. Let's wait for several days, maybe they will converge on something.

Reply:
I'm happy with either `логический` or `логический (Boolean)`.

Reply:
Back from there. Nothing exciting. The word `флаг` does not sound right.

There was one very interesting comment though. Somebody said that when they translated some MS products, one of the important goals was to make the translation comprehensible not only to people who do not speak English, but also to those who speak both Russian and English, so that they would not have to guess whether `логический` stood for `Boolean` or something else. On that premise, the person suggested `булев`, and noted that technical documentation is not a novel and its purpose is not to engage the user but to make the use of the product simpler.

This opinion helped me to make my mind. I decided on `логический (Boolean)`; somewhat verbose but clear.

Reply:
That's a thorough approach :)
Thanks, merged.